src/imitation/algorithms/mce_irl.py

```diff
@@ -45,6 +45,10 @@ def mce_partition_fh(
         \pi is a 3d array, indexed \pi[t,s,a].
     """
     # shorthand
+    # TODO ask Adam if this isn't better solved by changing the types in
```

Comment:
I agree this should be fixed upstream, I think @ianyfan is planning on working on this.

Reply:
Great, good to know for reference.